src/portfft/descriptor.hpp

```diff
@@ -650,6 +676,7 @@ class committed_descriptor {
    * @param num_global_level_dimensions number of global level dimensions in the committed size
    */
   void allocate_scratch_and_precompute_scan(Idx num_global_level_dimensions) {
```

Comment:
This function can also benefit from logging the inclusive scan precomputed

Reply:
done